### ISSUE TITLE
Feature request: keyboard shortcuts for next/previous window.

### ISSUE THREAD

**Issue by 0rion8** [iamlemec/miniview#20]
Can you add set option: keyboard shortcut for next window right alt+left arrow and previous window: right alt+right arrow.

**Comment by iamlemec** [comment]
Yup, definitely a good idea. Will work on this too. Thanks for the input!
